.github/workflows/up-to-date.yml

```diff
@@ -0,0 +1,25 @@
+name: Banch up-to-date with target
+
+on: pull_request
```

Comment:
Which pull_request events do you want this to fire on? `synchronize` maybe?

Reply:
Interesting. I had no idea `pull_request` resolved to so many different events. I'll look at this closer (and take a look at how this might be applied to other actions), thanks!